<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parametres</class>
 <widget class="QDialog" name="Parametres">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Paramètres</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>411</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Paramètres</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>371</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>SWAP</string>
    </property>
    <widget class="QSpinBox" name="spSwapStart">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spSwapEnd">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Echanger les éléments n° :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>50</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>et</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>SUM</string>
    </property>
    <widget class="QSpinBox" name="spSum">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sommer les</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> premiers elements</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>MEAN</string>
    </property>
    <widget class="QSpinBox" name="spMean">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Moyenner les</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>premiers elements</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Affichage</string>
    </property>
    <widget class="QSpinBox" name="spAffichage">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Affichers les</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>premiers elements de la pile</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="bOk">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>411</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Pile</string>
   </property>
   <widget class="QListWidget" name="pileHistorique">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>371</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
